web/src/containers/Requests.js

```diff
@@ -63,6 +63,7 @@ const Requests = () => {
           setError(null);
         }}>Okay</Button>
       </Box> }
+      <PendingRequests/>
```

Comment:
Is there a higher component this can be placed in? So its less dependant on the creating of requests?